Pipeline/PackageBuildOutputs/PackageBuildOutputs.groovy

```diff
@@ -84,123 +89,208 @@ BuildProperties.setProperty("dbb.file.tagging", "true") // Enable dbb file taggi
 // Map of last level dataset qualifier to DBB CopyToFS CopyMode.
 def copyModeMap = evaluate(props.copyModeMap)
 
-// read build report data
-println("** Read build report data from $props.workDir/BuildReport.json")
-def jsonOutputFile = new File("${props.workDir}/BuildReport.json")
-
-if(!jsonOutputFile.exists()){
-	println("** Build report data at $props.workDir/BuildReport.json not found")
-	System.exit(1)
-}
+// Hashmap of BuildOutput to Record
+Map<String, Record> buildOutputsMap = new HashMap<String, Record>() 
 
-def buildReport= BuildReport.parse(new FileInputStream(jsonOutputFile))
+// Field to store default tarFileLabel (buildInfo.label) when cli argument tarFileName is not passed.
+def String tarFileLabel = "Default"
 
-// finds all the build outputs with a deployType
-def executes= buildReport.getRecords().findAll{
-	try {
-		(it.getType()==DefaultRecordFactory.TYPE_EXECUTE || it.getType()==DefaultRecordFactory.TYPE_COPY_TO_PDS) &&
-				!it.getOutputs().isEmpty()
-	} catch (Exception e){}
-}
+// iterate over all build reports to obtain build output
+props.buildReportOrder.each{ buildReportFile -> 
+	println("** Read build report data from $buildReportFile")
+	def jsonOutputFile = new File(buildReportFile)
 
-if (props.deployTypeFilter){
-	println("** Filtering Output Records on following deployTypes: ${props.deployTypeFilter}")
-	executes.each {
-		// filtered executes
-		def filteredOutputs =  it.getOutputs().findAll{ o ->
-			o.deployType != null && (props.deployTypeFilter).split(',').contains(o.deployType)
-		}
-		// Manipulating the scope of build outputs
-		it.getOutputs().clear()
-		it.getOutputs().addAll(filteredOutputs)
+	if(!jsonOutputFile.exists()){
+		println("** Build report data at $buildReportFile not found")
+		System.exit(1)
```

Comment:
We should use standard: 
-> println("*! Error:  Build report data at $buildReportFile not found")